src/Workspaces/MSBuildTest/MSBuildWorkspaceTests.cs

```diff
@@ -1768,7 +1768,7 @@ public async Task TestParseOptions_CSharp_Compatibility_None()
         public async Task TestParseOptions_CSharp_LanguageVersion_Default()
         {
             CreateCSharpFiles();
-            await AssertCSParseOptionsAsync(CS.LanguageVersion.CSharp7, options => options.LanguageVersion);
+            await AssertCSParseOptionsAsync(CS.LanguageVersion.CSharp8, options => options.LanguageVersion);
```

Comment:
This should probably be some sort of "get effective version of LatestMajor or Default" or something. What's the test actually asserting?

Reply:
I read it as testing what the default version is. Do you regard the previous behavior of hard-coding 7.0 as a bug?

Reply:
Yes, that was probbaly a bug.

Reply:
And yes, I would expect you to fix this, since this would direct fallout of your design change here.

Reply:
OK, can do.